# frame_type: 0x38(56)
#등속구간에서만 가능
#운전중인 상태에서 목표 절대 위치값[pulse]의 변경을 요청
# 절대위치 ->  min: -134,217,728 max: 134,217,727
int32 target_abs_pos